cartography/intel/gcp/apigateway.py

```diff
@@ -332,6 +427,81 @@ def cleanup_apis(neo4j_session: neo4j.Session, common_job_parameters: Dict) -> N
    """
     run_cleanup_job('gcp_apigateway_apis_cleanup.json', neo4j_session, common_job_parameters)
 
+@timeit
+def load_apis_entity_relation(session: neo4j.Session, api: Dict, update_tag: int) -> None:
+    session.write_transaction(load_apis_entity_relation_tx, api, update_tag)
+
+
+@timeit
+def load_apis_entity_relation_tx(tx: neo4j.Transaction, api: Dict, gcp_update_tag: int) -> None:
+    """
+        :type neo4j_session: Neo4j session object
+        :param neo4j session: The Neo4j session object
+
+        :type apis: Dict
+        :param apis: A list of GCP APIs
+
+        :type project_id: str
+        :param project_id: Current Google Project Id
+
+        :type gcp_update_tag: timestamp
+        :param gcp_update_tag: The timestamp value to set our new Neo4j nodes with
+
+        :rtype: NoneType
+        :return: Nothing
+    """
+    ingest_entities = """
+    UNWIND {entities} AS entity
+    MATCH (principal) where principal.email = entity.email
```

Comment:
in the match statement, reference of the neo4j node is missing. something like `MATCH (principal:<<<>>>)`

Reply:
principal can be anything user, domain, service account. We are only assigning principal.email with the entity.email